deploy/runtime/ui/PV Albedo and Radiation.txt

```diff
@@ -1,27 +1,27 @@
 PV Albedo and Radiation
 882
-215
-7
+639
```

Comment:
This UI form is not loading in correctly for me: 

![image](https://user-images.githubusercontent.com/54449384/191786345-cf973950-3cea-4c64-b8fd-fc75d6bc9409.png)

Reply:
> Are there extra steps I need to take to get PV Albedo and Radiation.txt to work as a UI form? So far it is not being read properly. I will review in more detail when I can get the albedo UI form working.

Yes, sorry, please see above dependencies (i.e., the bifacial branches of wex and ssc).

Reply:
Forgot about wex. Thanks!